app/helpers/application_helper.rb

```diff
@@ -1,2 +1,119 @@
+require 'attendance'
+
 module ApplicationHelper
+  def present_marked?
+    @attendance = current_employee.todays_attendance_of_employee
+    !@attendance.blank?
```

Comment:
we can simply use "@attendance.present?"

Reply:
done, using a function 'present?' in Attendance model